README.md

```diff
@@ -13,3 +13,6 @@ Put files to be populated by Ignition in the tree rooted at
 Run the tool using `go run cmd/main.go`.
 
 The output ISO is written to `output/fleeting.iso`.
+
+Boot the ISO in a VM with at least 4096MiB of RAM. No storage is required.
```

Comment:
at least `16384`MiB of RAM

Found that with 4GB or 8GB, the assisted-image-service fails with no space on device. 
```
{"file":"/go/src/github.com/openshift/origin/main.go:66","func":"main.main.func1","level":"fatal","msg":"Failed to populate image store: failed to create minimal iso template for version map[cpu_architecture:x86_64 openshift_version:4.10 rootfs_url:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live-rootfs.x86_64.img url:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live.x86_64.iso version:410.84.202201251210-0]: write /data/isoutil2722569227/images/pxeboot/rootfs.img: no space left on device\n","time":"2022-02-23T21:44:45Z"}
```
Works fine with 16GB of RAM

Reply:
That's strange, I have definitely seen it work with 4096MiB RAM and no disk. There must be something else that determines how much space you get, but I can't think what.